.github/actions/git-diff-on-components/src/index.js

```diff
@@ -328,22 +333,23 @@ async function run() {
     const filesToBeCheckedByDependency = getFilesToBeCheckByDependency(componentsDependencies);
     const componentsThatNeedToBeModified = await getComponentsThatNeedToBeModified({ filesToBeCheckedByDependency, otherFiles });
     const componentsPendingForGitDiff = getComponentsPendingForGitDiff(componentsThatNeedToBeModified);
-    componentsDiffContents = await checkVersionModification(componentsPendingForGitDiff);
+    componentsDiffContents = await checkVersionModification(componentsPendingForGitDiff, componentsThatDidNotModifyVersion);
+  }
+
+  const totalErrors = componentsThatDidNotModifyVersion.length + componentsDiffContents.length;
+  let counter = 1;
+
+  if (componentsThatDidNotModifyVersion.length || componentsDiffContents.length) {
```

Comment:
Here you can also do:
```suggestion
  if (totalErrors) {
```